hw/dv/sv/sw_test_status_if/sw_test_status_if.core

```diff
@@ -0,0 +1,18 @@
+CAPI=2:
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+name: "lowrisc:dv:sw_test_status_if"
```

Comment:
```suggestion
name: "lowrisc:dv:sw_test_status"
```

Reply:
Done, with the requisite other renames of the directory etc.